### DISCUSSION TITLE
Cannot access http://domain/graphql on server (cloudpanel)

### DISCUSSION THREAD

**Discussion by HumzaAqil** [nuwave/lighthouse#2580] [Q&A]
### Issue Description:
Attempting to access https://domain/graphql/ and http://domain/graphql/ results in a 403 Forbidden error.
It is working fine locally (XAMPP) but when i upload my project on server and access it directly or via graphiql, this issue occurs.
### Steps to Reproduce:
1. Navigate to https://domain/graphiql/.
2. The following error is encountered:
   ```css
   Mixed Content: The page at 'https://domain/graphiql/' was loaded over HTTPS, but requested an insecure resource 'http://domain/graphql/'. This request has been blocked; the content must be served over HTTPS.


### Expected Behavior:
The GraphQL endpoint should be accessible without causing a mixed content error, and it should not result in a 403 Forbidden error.

### Actual Behavior:
Accessing the endpoint results in a 403 Forbidden error.
Mixed content error due to requesting an insecure resource from a secure page.
### Environment:
Server: CloudPanel
Web Server: Nginx
### Additional Context:
Ensure that the GraphQL endpoint is properly configured to be served over HTTPS and check the Nginx configuration to allow access to the endpoint.

PLEASE LET ME KNOW IF YOU NEED ANY OTHER INFO REGARDING THIS ISSUE.

**Comment by spawnia** [comment]
In `config/app.php`, ensure that `url` is set to a value that begins with `https://`.

**Reply by HumzaAqil** [reply]
'url' => env('APP_URL', 'https://mydomain.net'),

no luck, still 403 Forbidden nginx :(
(i cannot reveal my domain as it belongs to my company)

**Reply by spawnia** [reply]
`APP_URL` could still overwrite that value to be non-https.

**Reply by HumzaAqil** [reply]
it is also set to the same domain on https

**Reply by spawnia** [reply]
I have no other ideas on how to fix this.
